MekHQ/src/mekhq/gui/adapter/PersonnelTableMouseAdapter.java

```diff
@@ -968,7 +968,7 @@ public void actionPerformed(ActionEvent action) {
                 break;
             }
             case CMD_ADD_LOG_ENTRY: {
-                AddOrEditPersonnelEntryDialog addPersonnelLogDialog = new AddOrEditPersonnelEntryDialog(gui.getFrame(), true, gui.getCampaign().getDate()); //$NON-NLS-1$
+                AddOrEditPersonnelEntryDialog addPersonnelLogDialog = new AddOrEditPersonnelEntryDialog(gui.getFrame(), true, gui.getCampaign(), gui.getCampaign().getLocalDate());
```

Comment:
Why are we passing in the campaign date when we're already passing in the campaign?

Reply:
Personal stylistic choice, I've never been much of a fan of overloaded methods and would rather add more parameters.

Additionally, we now require both the campaign and the date, plus it is not guaranteed that the date will be the campaign date.